content/arduino-cloud/09.business/02.aws-s3-exporter/content.md

```diff
@@ -0,0 +1,465 @@
+
+Other settings can remain at their **default values** unless specific customizations are needed.
+
+After defining the required settings, proceed to **Submit** the bucket creation. Once successfully created, the bucket will be listed under **General purpose buckets**.
+
+This process creates the **temporary bucket** to store the **Lambda binaries** and the **CloudFormation template (CFT)**. The assigned bucket name is **lambdas3binaries** in this example.
+
+![Lambda Binary & CFT Bucket](assets/s3_bucket_lambdas3bin.png)
+
+#### Temporary Bucket
+
+The **lambdas3binaries** bucket is used to store the required deployment files. The following files must be uploaded to this bucket:
+
+- [Lambda binaries](https://github.com/arduino/aws-s3-integration/releases) (`.zip` file)
+- [CloudFormation template (CFT)](https://github.com/arduino/aws-s3-integration/releases) (`.yaml` file)
+
+To upload the files, navigate to **Amazon S3** and open the **`lambdas3binaries`** bucket. The available options for managing the bucket will be displayed:
+
+![lambdas3binaries Bucket Setup (1)](assets/s3_bucket_lambdas3bin_setup_1.png)
+
+Select the **Upload** option within the **Objects** panel.
+
+Manually upload the required files by either using the file browser to select the `.zip` and `.yaml` files or dragging and dropping them into the designated upload area. Once the files are recognized, the screen should resemble the following image:
+
+![lambdas3binaries Bucket Setup (2)](assets/s3_bucket_lambdas3bin_setup_2.png)
+
+Click the **Upload** button to begin the upload. Once the upload is complete, a confirmation screen will be displayed:
+
+![lambdas3binaries Bucket Setup (3)](assets/s3_bucket_lambdas3bin_setup_3.png)
+
+After a successful upload, the files will be listed under the **Objects** tab within the **`lambdas3binaries`** bucket:
+
+![lambdas3binaries Bucket Setup (4)](assets/s3_bucket_lambdas3bin_setup_4.png)
+
+#### CSV Destination Bucket
+
+A second bucket needs to be created following the same process as the [Temporary bucket](#temporary-bucket). This bucket will be the **CSV destination bucket**, where all generated CSV files will be stored. It is important to make sure this bucket is created in the same AWS region where the CloudFormation stack will be deployed.
+
+Navigate to the **Amazon S3** service and select **Create bucket**. In the bucket creation window, specify the bucket name and check that the same AWS region is selected.
+
+Keep the recommended default settings for **Object Ownership** and **Public Access** to maintain security compliance. Once all settings are verified, proceed with the bucket creation.
+
+![lambdas3binaries Bucket Setup (1)](assets/s3_bucket_csvdests3int_1.png)
+
+After the bucket has been successfully created, it will be listed among the available S3 buckets. Select the newly created **CSV destination bucket** to proceed with additional configurations if necessary.
```

Comment:
```suggestion
After the bucket has been successfully created, it will be listed among the available S3 buckets. If necessary, you can select the newly created **CSV destination bucket** to proceed with additional configurations.
```